src/Bicep.Cli/Program.cs

```diff
@@ -126,6 +127,7 @@ private ServiceProvider ConfigureServices()
                 .AddSingleton<IFileSystem, FileSystem>()
                 .AddSingleton<IConfigurationManager, ConfigurationManager>()
                 .AddSingleton<ITokenCredentialFactory, TokenCredentialFactory>()
+                .AddSingleton<ApiVersionProvider, ApiVersionProvider>()
```

Comment:
This should be .AddSingleton<IApiVersionProvider, ApiVersionProvider>()?

Reply:
fixed

Reply:
Reverted, not working